x/examples/website-wrapper-app/.scripts/build_mobileproxy.sh

```diff
@@ -16,9 +16,15 @@
 
 PLATFORM="$1"
 TAG="${2:-"x/v0.0.1"}"
+OUTPUT="${3:-output}"
 
-git clone --depth 1 --branch "${TAG}" https://github.com/Jigsaw-Code/outline-sdk.git output/outline-sdk
-cd output/outline-sdk/x
+if [ "$OUTPUT" = "/" ] || [ "$OUTPUT" = "*" ]; then
+  echo "Error: OUTPUT cannot be '/' or '*'. These are dangerous values."
+  exit 1
+fi
+
+git clone --depth 1 --branch "${TAG}" https://github.com/Jigsaw-Code/outline-sdk.git $OUTPUT/outline-sdk
```

Comment:
You need to properly quote the variables:
```suggestion
git clone --depth 1 --branch "${TAG}" https://github.com/Jigsaw-Code/outline-sdk.git "${OUTPUT}/outline-sdk"
```

Please run shellchecker. There's a number of small things to fix: https://www.shellcheck.net/

You can copy what we do in the outline-server repo: https://github.com/search?q=repo%3AJigsaw-Code%2Foutline-server%20shellcheck&type=code, especially the rc file